packages/frontend/component/src/ui/menu/desktop/controller.ts

```diff
@@ -18,7 +18,25 @@ export const useMenuContentController = ({
   const [open, setOpen] = useState(defaultOpen ?? false);
   const actualOpen = controlledOpen ?? open;
   const contentSide = side ?? 'bottom';
-  const [contentOffset, setContentOffset] = useState<number>(0);
+  const [contentOffset, setContentOffsetO] = useState<number>(0);
+
+  const setContentOffset = useCallback(
```

Comment:
Can we avoid decimals from the source by rounding the getBoundingClientRect result?

Reply:
I tried this but it will resulting in more shaking (0/1/0/1 etc)